### DISCUSSION TITLE
iOS崩溃 版本4.3.3

### DISCUSSION THREAD

**Discussion by cchvip** [Tencent/libpag#1667] [Q&A] [answered]
## 【版本信息】

4.3.3

## 【平台信息】

 iOS 原生

设备信息：iPhone 6s Plus
![image](https://github.com/Tencent/libpag/assets/16414128/8a99c023-ac75-4229-9a52-128cc61bd88f)

![image](https://github.com/Tencent/libpag/assets/16414128/ffdec873-1b66-4c76-8112-6ceee64ec638)

**Comment by kevingpqi123** [comment, marked as answer]
这个问题其它业务也有报过，量比较少，之前分析暂时没有结论，可以看下能否找到复现路径，有复现路径解决起来就会快很多

**Reply by cchvip** [reply]
只能捞到对应的资源，但是正常播放测试也没问题。4.3.3的其他异常崩溃非常多。已经把版本回退到4.2.88。崩溃相对较少
